src/applications/facility-locator/components/ResultsList.jsx

```diff
@@ -115,8 +122,20 @@ class ResultsList extends Component {
           className="search-result-title facility-result"
           ref={this.searchResultTitle}
         >
-          We’re sorry. We couldn’t complete your request. Please refresh the
-          page or try again later.
+          <p>We’re sorry. We couldn’t complete your request.</p>
+          <p>
+            Please try again in a few minutes. Or, if you need care right away
+            for a minor illness or injury, search for your nearest VA health
+            facility or find{' '}
+            <a href="https://vaurgentcarelocator.triwest.com/">
```

Comment:
Is there a reason this link doesn't open in a new tab?

Reply:
The reason here is that I forgot to add the target/rel attributes 😆 Updating!